kpi/views/v2/data.py

```diff
@@ -556,9 +556,11 @@ def duplicate(self, request, pk, *args, **kwargs):
         deployment = self._get_deployment()
         # Coerce to int because back end only finds matches with same type
         submission_id = positive_int(pk)
+        original_submission = deployment.get_submission(submission_id, request.user)
         duplicate_response = deployment.duplicate_submission(
             submission_id=submission_id, user=request.user
         )
+        deployment.copy_submission_extras(original_submission['_uuid'], duplicate_response['_uuid'])
```

Comment:
Let's try to stay under the 80 characters limit.

Reply:
Ok